pyproject.toml

```diff
@@ -149,8 +150,6 @@ pytest-timeout = ">=1.0.0"
 pytest-xdist = ">=1.14"
 "repoze.debug" = ">=1.0.2"
 
```

Comment:
if this goes away, it may need some compensating work in some make situation because:
* `wheel` is not installed by default
* `wheel` does not show up in `pip freeze` even if installed, so it's hard to see that it's the key needed component (note that it does appear in `pip list` in this case)
* I _think_ the absence of `wheel` means some operations with github fail to included essential components and i think results in our thinking something is not available when it is

Reply:
Oh, never mind. As you explained in Slack, @willronchetti, this dependency only got moved. There may still be an issue where it's loading the wrong version, but not due to this PR.